packages/ui-kit/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,67 @@
+import { useButton } from '@mui/base/useButton'
+import classnames from 'classnames'
+import * as React from 'react'
+import { useForwardedRefCallback } from '../../helpers'
+import { DefaultThemes, ThemeStateProps, useSetupTheme } from '../ThemeProvider'
+import componentStyles from './Button.module.scss'
+
+export interface ButtonProps extends React.ComponentPropsWithoutRef<'button'> {
+  overridable?: boolean
```

Comment:
What is `overridable` for?

Reply:
We marked with the tag `devOnly` stories that should be visible in Storybook to developers (us) but hidden in the production version of the Storybook. Also, we have stories that we're using in docs but not directly as a component's story, e.g. `TimeSeries`'s [styling example](https://storybook.propeldata.com/?path=/docs/customization-components--overview#styles) on Customization/Components page but not among `TimeSeries`'s [stories](https://storybook.propeldata.com/?path=/docs/components-timeseries--overview). Previously, we marked such stories with `pattern` tag, but I've renamed it to `hidden`.

Reply:
`overridable ` is a part of the [following upcoming feature](https://github.com/propeldata/ui-kit/pull/206#discussion_r1581013464).